dataengineeringutils3/pulumi.py

```diff
@@ -0,0 +1,99 @@
+def check_business_unit(business_unit: str):
+    """Checks if business_unit is valid.
+
+    Parameters
+    ----------
+    business_unit : str
+        The business unit of the team that owns the resources. This should be one of
+        "HQ", "HMPPS", "OPG", "LAA", "HMCTS", "CICA", "Platforms".
+
+    Raises
+    ------
+    ValueError
+        If the value of business_unit is valid a ValueError will be raised.
+    """
+    if business_unit not in [
+        "HQ",
+        "HMPPS",
+        "OPG",
+        "LAA",
+        "HMCTS",
+        "CICA",
+        "Platforms",
+    ]:
+        raise ValueError(
+            "business_unit must be one of HQ, HMPPS, OPG, LAA, HMCTS, CICA, or"
+            "Platforms"
```

Comment:
Missing end-of-line space between 'or' and 'Platforms'

Reply:
Fixed in f4be262.